modules/virt-hot-unplugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-unplugging-a-virtual-disk-using-the-web-console{context}"]
+= Hot-unplugging a virtual disk using the web console
+
+Hot-unplug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* Your virtual machine must be running with a hot-plugged disk attached.
+
+.Procedure
+
+. Click *Workloads → Virtualization → Virtual Machines* from the side menu.
```

Comment:
Same comment as the other procedure. This should be:

. Click *Workloads* → *Virtualization* from the side menu.
. On the *Virtual Machines* tab, select the running virtual machine with the disk that you want to hot-unplug.
. Click the *Disks* tab.